cypress/plugins/testDataProcessor.js

```diff
@@ -108,9 +108,11 @@ function processSetResponseJson(setResponseJsonData) {
  *  getErrorContentObjectJson()
  */
 function getErrorContentObjectJson() {
-  const errorSchemaJson = fetchDataFromFile(CONSTANTS.ERROR_CONTENT_OBJECTS_PATH);
-  if (errorSchemaJson) {
-    return errorSchemaJson;
+  const errorContentJson = fetchDataFromFile(CONSTANTS.ERROR_CONTENT_OBJECTS_PATH);
+  const externalErrorSchemaJson = fetchDataFromFile(CONSTANTS.EXTERNAL_ERROR_CONTENT_OBJECTS_PATH);
+  const combinedErrorContentJson = Object.assign(errorContentJson, externalErrorSchemaJson);
+  if (combinedErrorContentJson) {
+    return combinedErrorContentJson;
```

Comment:
what if we dont have external file

Reply:
It fail with error "Unable to find Error content validation object for CUSTOM_ERROR" and if the key present in fcs means it will end at schema validation

Reply:
It will fail if we don't have the external file?

Reply:
yes, confirmed that it wont fail.